wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -459,6 +471,7 @@ def _add_to_launch_queue(self, run: SweepRun) -> bool:
             author=self._kwargs.get("author"),
         )
         run.queued_run = queued_run
+        run.state = RunState.RUNNING  # assume it will get picked up
```

Comment:
Ah, this would be a lot cleaner if we've unified queued runs and runs. Not slated for Q2, but maybe a TODO indicating to fix once we've done that would be good